src/Symfony/Bundle/DebugBundle/DependencyInjection/Configuration.php

```diff
@@ -53,6 +54,19 @@ public function getConfigTreeBuilder()
             ->end()
         ;
 
+        if (method_exists(HtmlDumper::class, 'setTheme')) {
+            $rootNode
+                ->children()
+                    ->enumNode('theme')
+                        ->info('Changes the color of the dump() output when rendered directly on the templating. "dark" (default) or "light"')
+                        ->example('"dark"')
```

Comment:
The other `example` calls we have do not wrap strings, should probably be `->example('dark')`.